src/components/CytoscapeGraph/graphs/GraphStyles.ts

```diff
@@ -18,15 +18,10 @@ import * as Cy from 'cytoscape';
 
 export const DimClass = 'mousedim';
 
-// UX-specified colors, widths, etc
-const ColorDanger = PfColors.Danger; // Same as Health.FAILURE.color
-const ColorHealthy = PfColors.Success; // Same as Health.HEALTHY.color
-const ColorWarning = PfColors.Warning; // Same as Health.DEGRADED.color
-
-const EdgeColor = ColorHealthy;
+let EdgeColor: string;
 const EdgeColorDead = PfColors.Black500;
-const EdgeColorDegraded = ColorWarning;
-const EdgeColorFailure = ColorDanger;
+let EdgeColorDegraded: string;
```

Comment:
It would be nice if we had a little stronger typing than just `string` for these colors. Even though it can just be a type alias for string. It aids in the documentation and prevents accidental invalid usage.